client/app/bundles/shared/startup/registration.jsx

```diff
@@ -0,0 +1,9 @@
+// @flow
+import ReactOnRails from 'react-on-rails';
+
+import Logo from '../components/Logo';
+
+// This is how react_on_rails can see the Components in the browser.
+ReactOnRails.register({
+  Logo,
```

Comment:
Can get rid of trailing comma here

Reply:
I think Code Climate fails without the trailing comma, unless we put this on a single line, though I figure we will have additional components in the future. I do want to have a separate task of simplifying the folder hierarchy with component registration, but will need some time to think about how this can be achieved.